xrpl/models/base_model.py

```diff
@@ -213,6 +213,30 @@ def _from_dict_single_param(
             )
         raise XRPLModelException(error_message)
 
+    @classmethod
+    def process_xrpl_json(
```

Comment:
```suggestion
    def _process_xrpl_json(
```

Looks like this is a private method so we should prefix it with underscore `_`

Reply:
thanks, I've addressed it. I had to make changes in two other function call places, introduced a separate commit